rel-eng/openshift-master.service

```diff
@@ -2,6 +2,11 @@
 Description=OpenShift Master
 Documentation=https://github.com/openshift/origin
 After=network.target
+After=etcd.service
+Wants=etcd.service
```

Comment:
It will log that they're not found and move on unless it's a Requires, in which case it would prevent the service from starting.